docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -1,483 +1,184 @@
-+
-[source,shell]
-----
-kubectl -n syn port-forward svc/argocd-server 8443:443
+kubectl --context k3d-steward -n syn expose deployment argocd-server --type=LoadBalancer --port=8080 --name=argocd-server-lb --overrides='{"metadata":{"labels":{"argocd.argoproj.io/instance":null}}}'
 ----
 
-With these steps, the local `k3s` or `docker-desktop` instance is now Syn enabled, has Argo CD running and automatically syncs the manifests found in the cluster catalog Git repository which was generated by Commodore and is stored in GitLab under https://${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1/.
+Open Argo CD in your browser at http://localhost:35778. Log in to Argo CD with the username `admin` and the password returned by this command:
 
-[TIP]
-====
-If you want or need to reset the bootstrap token, this is the way to go:
-Get the Kubernetes API URL with `kubectl cluster-info` and replace `REPLACE_API_URL` in the command below:
-[source,shell]
+[source,bash]
 ----
-curl -k -H "${LIEUTENANT_AUTH}" -H "Content-Type: application/json-patch+json" -X PATCH -d '[{ "op": "remove", "path": "/status/bootstrapToken" }]' "REPLACE_API_URL/apis/syn.tools/v1alpha1/namespaces/lieutenant/clusters/${CLUSTER_ID}/status"
+kubectl --context k3d-steward -n syn get secret steward -o json | jq -r .data.token | base64 --decode
 ----
-====
 
-== Cleaning Up
+After these steps, the local K3s cluster is now Syn enabled, and a Git repository called "Project Syn Cluster" is available in GitLab. It's worth pointing out that Argo CD hasn't yet synchronized any changes, and the GitLab repository is empty.
 
-Once you've gone through all these steps, you can cleanup all generated stuff using the following steps:
+== What's Next?
 
-. Delete the `Cluster` object
-+
-[source,shell]
-----
-kubectl -n lieutenant delete cluster ${CLUSTER_ID}
-----
-This will also delete the associated `GitRepo` object and with that the cluster configuration file in the tenant configuration repository and the cluster catalog Git repository on GitLab.
+This guide provided a quick overview of Lieutenant and Steward. If you are interested in knowing more about Project Syn, proceed to the xref:tutorials/getting-started-2.adoc[second part] to learn about Commodore, the third major component of Project Syn.
+
+Please don't execute the steps in the following "Cleaning Up" section if you intend to perform the actions of the xref:tutorials/getting-started-2.adoc[second part] of this guide. We will reuse the clusters in their current state.
+
+== Cleaning Up
+
+If you don't wish to proceed to the xref:tutorials/getting-started-2.adoc[second part] of this tutorial, you can clean up all generated clusters and GitLab projects using the following steps:
 
-. Delete the `Tenant` object
+. Delete the cluster and tenant objects, removing the GitLab projects at the same time:
```

Comment:
Might be worth to add a Danger admonition, warning yet again to only execute if no intention to do the second part.

Reply:
It's indicated above.

Reply:
Yes, I know. But you know how things are more than 240 characters… That is too much to remember such an important thing 😛.